File: Android/AdventourAndroidVersion/app/src/main/java/com/adventour/android/Location.java
package com.adventour.android;

public class Location
{
    private String fsq_id;
    private String name;
    private String description;
    private String tel;
    private String website;
    private String rating;

    public Location(String fsq_id, String name, String description, String tel, String website, String rating) {
        this.fsq_id = fsq_id;
        this.name = name;
        this.description = description;
        this.tel = tel;
        this.website = website;
        this.rating = rating;
    }

    public String getFsq_id() {
        return fsq_id;
    }

    public void setFsq_id(String fsq_id) {
        this.fsq_id = fsq_id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public String getTel() {
        return tel;
    }

    public void setTel(String tel) {
        this.tel = tel;
    }

    public String getWebsite() {
        return website;
    }

    public void setWebsite(String website) {
        this.website = website;
    }

    public String getRating() {
        return rating;
    }

    public void setRating(String rating) {
        this.rating = rating;
    }
}

File: iOS/ios/ios/Location.swift
//
//  Location.swift
//  ios
//
//  Created by Glenn Hartwell on 10/4/22.
//

import Foundation

class Location {
    
    private var fsq_id: String
    private var name: String
    private var description: String
    private var tel: String
    private var website: String
    private var rating: Double
    private var popularity: Double
    private var price: Double
    
    init(fsq_id: String, name: String, description: String, tel: String, website: String, rating: Double, popularity: Double, price: Double) {
        self.fsq_id = fsq_id
        self.name = name
        self.description = description
        self.tel = tel
        self.website = website
        self.rating = rating
        self.popularity = popularity
        self.price = price
    }
    
    func getFsqId() -> String {return fsq_id}
    func getName() -> String {return name}
    func getDescription() -> String {return description}
    func getTel() -> String {return tel}
    func getWebsite() -> String {return website}
    func getRating() -> Double {return rating}
    func getPopularity() -> Double {return popularity}
    func getPrice() -> Double {return price}
}
